spec/rspec/rails/matchers/route_to_spec.rb

```diff
@@ -13,50 +13,50 @@ def assert_recognizes(*)
   end
 
   it "delegates to assert_recognizes" do
-    expect(self).to receive(:assert_recognizes).with({"these" => "options"}, {method: :get, path: "path"}, {})
-    expect({get: "path"}).to route_to("these" => "options")
+    expect(self).to receive(:assert_recognizes).with({ "these" => "options" }, { method: :get, path: "path" }, {})
+    expect({ get: "path" }).to route_to("these" => "options")
   end
 
   context "with shortcut syntax" do
     it "routes with extra options" do
-      expect(self).to receive(:assert_recognizes).with({controller: "controller", action: "action", extra: "options"}, {method: :get, path: "path"}, {})
+      expect(self).to receive(:assert_recognizes).with({ controller: "controller", action: "action", extra: "options" }, { method: :get, path: "path" }, {})
       expect(get("path")).to route_to("controller#action", extra: "options")
     end
 
     it "routes without extra options" do
       expect(self).to receive(:assert_recognizes).with(
-        {controller: "controller", action: "action"},
-        {method: :get, path: "path"},
+        { controller: "controller", action: "action" },
+        { method: :get, path: "path" },
         {}
       )
       expect(get("path")).to route_to("controller#action")
     end
 
     it "routes with one query parameter" do
       expect(self).to receive(:assert_recognizes).with(
-        {controller: "controller", action: "action", queryitem: "queryvalue"},
-        {method: :get, path: "path"},
-        {'queryitem' => 'queryvalue'}
+        { controller: "controller", action: "action", queryitem: "queryvalue" },
+        { method: :get, path: "path" },
+        { 'queryitem' => 'queryvalue' }
       )
       expect(get("path?queryitem=queryvalue")).to route_to("controller#action", queryitem: 'queryvalue')
     end
 
     it "routes with multiple query parameters" do
       expect(self).to receive(:assert_recognizes).with(
-        {controller: "controller", action: "action", queryitem: "queryvalue", qi2: 'qv2'},
-        {method: :get, path: "path"},
-        {'queryitem' => 'queryvalue', 'qi2' => 'qv2'}
```

Comment:
Maybe disable this cop, too? This code looks good with or without spaces.

Reply:
I think consistency wins on this Cop.  This is a cop that's enabled on the base, and unlike the others which are necessary to support older Rubies, this is a style issue.  So either disable it on the base (which is fine) or make these changes.